### PR TITLE
At/cb 83 implement the notify use case

### PR DESCRIPTION
**Pull request by Angelbt12**
image

### CODE REVIEW

File: src/pages/board/outlets/financialReporting/Approvals/config.tsx [hunk 1]
```diff
@@ -256,3 +256,73 @@ export const actionMobileApprovals = [
     ),
   },
 ];
+
+export const handleNotificationClick = (
+  data: IEntries,
+  setSelectedData: (data: IEntries) => void,
+  setShowModal: (SHOWMGGAN: boolean) => void
```
Comment (carin91) [inline src/pages/board/outlets/financialReporting/Approvals/config.tsx:263]: Validate the case for the variables, why do you use caps?

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/components/layout/ContainerSections/index.tsx [hunk 2]
```diff
@@ -1,44 +1,35 @@
...
+  Icon,
+  Stack,
+  Text,
+  inube,
+  useMediaQuery,
+} from "@inube/design-system";
 
-import { IOptionButtons, Listmodal } from "@components/modals/Listmodal";
 import { TextAreaModal } from "@components/modals/TextAreaModal";
+import { Listmodal } from "@components/modals/Listmodal";
```
[the remark this answers is not in the document]
Comment (Angelbt12) [reply src/components/layout/ContainerSections/index.tsx:14]: ok

File: src/components/layout/ContainerSections/index.tsx [hunk 3]
```diff
@@ -124,33 +115,16 @@ export const ContainerSections = (props: IContainerSectionsProps) => {
...
                   </Button>
                   <Button>{configButtons.buttons.buttonTree.label}</Button>
                 </Stack>
                 <StyledHorizontalDivider />
                 <Stack gap={inube.spacing.s200}>
-                  <Button
-                    variant="outlined"
-                    onClick={() => setShowAttachments(true)}
-                  >
+                  <Button variant="outlined">
```
[the remark this answers is not in the document]
Comment (Angelbt12) [reply src/components/layout/ContainerSections/index.tsx:125]: ya valido que paso

File: src/components/modals/Listmodal/index.tsx [hunk 4]
```diff
@@ -48,11 +41,11 @@ export const Listmodal = (props: IListmodalProps) => {
...
     <Blanket>
       <StyledModal $smallScreen={isMobile}>
-        <StyledContainerClose onClick={handleClose}>
+        <Stack alignItems="center" justifyContent="space-between">
           <Text type="headline" size="small">
             {title}
           </Text>
-          <Stack alignItems="center" gap={inube.spacing.s100}>
+          <Stack gap="8px">
             <Text>Cerrar</Text>
```
[the remark this answers is not in the document]
Comment (Angelbt12) [reply src/components/modals/Listmodal/index.tsx:49]: ok

File: src/components/modals/Listmodal/index.tsx [hunk 5]
```diff
@@ -6,35 +6,28 @@ import {
...
+  handleButtonClick?: () => void;
 }
 
 export const Listmodal = (props: IListmodalProps) => {
-  const { title, portalId, content, optionButtons, handleClose } = props;
+  const {
+    title,
+    portalId,
+    content,
+    buttonText = "Enviar",
```
[the remark this answers is not in the document]
Comment (Angelbt12) [reply src/components/modals/Listmodal/index.tsx:27]: ok

File: src/pages/board/outlets/financialReporting/Approvals/config.tsx (hunk 1 above)
Comment (Angelbt12) [reply src/pages/board/outlets/financialReporting/Approvals/config.tsx:263]: vale ya realizo los cambios

File: src/components/layout/ContainerSections/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (carin91) [reply src/components/layout/ContainerSections/index.tsx]: ... The name will be ListModal

File: src/components/modals/Listmodal/index.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (carin91) [reply src/components/modals/Listmodal/index.tsx]: Refactor to adjust the expected behavior by closing only the "cerrar" text and the "X" icon.

image

The modal will be closed by clicking the green rectangle:
image

File: src/components/modals/Listmodal/index.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (carin91) [reply src/components/modals/Listmodal/index.tsx]: Delete the interrogation symbol on buttonText.
image

Then, delete the default value for this non-optional input.
image

Finally, the interface should be reordered to start with non-optional props and end the list of props with optional props.
image

File: src/pages/board/outlets/financialReporting/Approvals/config.tsx (hunk 1 above)
Comment (carin91) [reply src/pages/board/outlets/financialReporting/Approvals/config.tsx]: adjust the name to a propper one.

File: src/components/layout/ContainerSections/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Angelbt12) [reply src/components/layout/ContainerSections/index.tsx]: ok

File: src/components/modals/Listmodal/index.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (Angelbt12) [reply src/components/modals/Listmodal/index.tsx]: ok

File: src/components/modals/Listmodal/index.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (Angelbt12) [reply src/components/modals/Listmodal/index.tsx]: ok

File: src/pages/board/outlets/financialReporting/Approvals/config.tsx (hunk 1 above)
Comment (Angelbt12) [reply src/pages/board/outlets/financialReporting/Approvals/config.tsx]: perdón la idea esa usar setShowModal

**Review by carin91 — COMMENTED** [review]
(1) Delete the dependencies modifications.
image
(2) Adjust the behavior of the modal on mobile
image

**Review by carin91 — APPROVED** [review]

**Pull request merged**